R/source.R

```diff
@@ -197,6 +210,21 @@ get_dynlib_ext <- function() {
   .Platform$dynlib.ext
 }
 
+get_specific_target_name <- function() {
+  sysinf <- Sys.info()
+
+  if  (!is.null(sysinf) && sysinf["sysname"] == "Windows") {
+    if (R.version$arch == "x86_64")
+      return("x86_64-pc-windows-gnu")
+
+    if (R.version$arch == "i386")
+      return("i686-pc-windows-gnu")
```

Comment:
Here and above: Please enclose the `return()` statement into curly braces, as described here: https://style.tidyverse.org/syntax.html#inline-statements